sound/soc/sof/topology.c

```diff
@@ -1277,9 +1320,6 @@ static int sof_widget_load_pga(struct snd_soc_component *scomp, int index,
 	/* get volume control */
 	scontrol = sm->dobj.private;
 
-	/* set cmd for pga kcontrol */
-	scontrol->cmd = SOF_CTRL_CMD_VOLUME;
-
```

Comment:
why is this removed?
and is this is correct, why is there still a volume allocated?

Reply:
@plbossart this is removed because I moved the command setting part to load_control(). Since both switch/volume controls are the same type of control with a difference in their max values, it makes sense to do it in load_control().

I havent added support for more than 1 kcontrol for pga comp so it still can have only 1 kcontrol that should be a volume kcontrol. So we will not allocate a volume table for switch controls.